tailwind.config.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
     './src/**/*.tsx',
     './index.html',
   ],
-  //Any class that is generated dynamically goes here
+  //Any class that's generated dynamically goes here
```

Comment:
why..?

Reply:
Oh interesting, I wonder if the spelling plugin I have installed did it.